media/js/firefox/family-nav.js

```diff
@@ -103,9 +103,6 @@ if (typeof window.Mozilla === 'undefined') {
 
         // public initialization point
         var _init = function() {
-            // all external adjunct nav links open in new tab
-            $adjunctNavs.find('a[rel="external"]').attr('target', '_blank');
```

Comment:
Because this variable is no longer being used, you should remove the definition also (as per the JS lint error)